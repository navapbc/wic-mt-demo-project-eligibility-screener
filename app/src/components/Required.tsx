import { ReactElement } from 'react'

export const Required = (): ReactElement => {
  return <abbr className="usa-hint usa-hint--required"> *</abbr>
}

export default Required
