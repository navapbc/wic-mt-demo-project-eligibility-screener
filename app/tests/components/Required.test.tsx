import Required from '@components/Required'

import { testSnapshot } from '../helpers/sharedTests'

it('should match snapshot', () => {
  testSnapshot(<Required />)
})
