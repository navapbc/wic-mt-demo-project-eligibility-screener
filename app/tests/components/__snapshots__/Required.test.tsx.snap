// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div>
  <abbr
    class="usa-hint usa-hint--required"
  >
     *
  </abbr>
</div>
`;
